juju/juju
# allow reload-spaces to add space, availability zone and vlan tag data to existing subnets

## Description of change

Allow the juju reload-spaces cmd to add space, availability zone and vlan tag data to 
existing subnets.  With this change you cannot change existing subnet data, only add.

## QA steps

Easiest if you have a MAAS config where config changes are possible.

1. juju bootstrap maas
2. With an existing subnet containing no space or vlan tag data, add both.
3. juju reload-spaces
3. juju subnets; verify the subnet now has a space named instead of "".
3. juju spaces; verify the new space is mapped to the correct subnet.
3. check the juju db for the vlan tag change.  db.subnets.find()
3. Change the space and vlan tag on the same subnet in Maas.
3. juju reload-spaces.
4. no change to data should be seen in juju subnets, nor juju spaces.
